timesketch/frontend-ng/src/components/Scenarios/Question.vue

```diff
@@ -0,0 +1,79 @@
+<!--
+Copyright 2022 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <div>
+    <div :class="highlightColor">
+      <div class="pa-2 pl-4" @click="expanded = !expanded" style="cursor: pointer">
+        <v-menu open-on-hover offset-y :open-delay="1000" :close-on-content-click="false">
+          <template v-slot:activator="{ on, attrs }">
+            <span style="font-size: 0.9em" v-bind="attrs" v-on="on">{{ question.display_name }}</span>
+          </template>
+          <v-card style="font-size: 0.9em" class="pa-3" width="400">
+            {{ question.description }}
+          </v-card>
+        </v-menu>
+      </div>
+
+      <div v-show="expanded">
+        <div style="font-size: 0.9em" class="pa-4">
+          <v-textarea disabled outlined flat hide-details auto-grow rows="3">
+            <template v-slot:prepend-inner>
+              <v-avatar color="grey" class="mt-n1" size="28"></v-avatar>
+            </template>
+          </v-textarea>
+          <v-card-actions class="pl-0">
+            <v-btn disabled x-small outlined color="primary"> Yes </v-btn>
```

Comment:
Is there an option to not use whitespaces before and after (I guess to make the buttons wider?

Reply:
e.g. adding a css to the v-btn? I am concerned moving forward some will apply a whitespace before and after the text, some will forget.

Reply:
This will not infact add any whitespace. It will be trimmed on render. This comes from the auto formatter we are using.